pallets/crowdloan-claim/src/mock.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
```

Comment:
Why not having a "Centrifuge" header for our modules?

```rust
// Copyright 2021 Centrifuge GmbH (centrifuge.io).
// This file is part of Centrifuge chain project.

// Centrifuge is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version (see http://www.gnu.org/licenses).

// Centrifuge is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
```

Reply:
Yes, it would like that to be default for our files. Although, I am not sure, which license we want to use.